cargo-pgx/src/command/install.rs

```diff
@@ -63,7 +63,10 @@ impl CommandExecute for Install {
             Some(config) => PgConfig::new_with_defaults(PathBuf::from(config)),
         };
         let pg_version = format!("pg{}", pg_config.major_version()?);
-        let profile = CargoProfile::from_flags(self.release, self.profile.as_deref())?;
+        let profile = CargoProfile::from_flags(
+            self.profile.as_deref(),
+            self.release.then_some(CargoProfile::Release).unwrap_or(CargoProfile::Dev),
```

Comment:
I argued with myself about continuing to push down the boolean for "self.release" instead, but I kinda like this as it's a little more explicit about the profile each command wants.  But I have no strong preference.

Reply:
I am happiest when booleans go away, yes.